apps/website/src/content/docs/progressindicator.mdx

```diff
@@ -1,29 +1,90 @@
 ---
 title: Progress indicator
-description: Progress indicators express an unspecified wait time or display the length of a process.
+description: Progress indicators express the length of in-progress operations.
 thumbnail: ProgressIndicator
 ---
 
 <p>{frontmatter.description}</p>
 
-## Progress linear
+## Usage
 
-<Placeholder componentPageName='progresslinear--determinate' />
+There are two types of progress indicators which are linear and radial. Linear indicators show progress on a linear bar, while radial indicators display progress in a circular form.
+
+The default progress indicator is indeterminate. If a [value](#value) is passed, the indicator becomes determinate.
 
 <LiveExample src='ProgressLinear.main.jsx' truncate={false}>
```

Comment:
Could some of these examples be combined? The demos are taking up too much space.

I'm thinking `ProgressLinear.main` and `ProgressRadial.main` could be combined. Similarly, `ProgressLinear.value` and `ProgressRadial.value` could be combined.

Reply:
I have combined the examples. There are two problems I encountered:
- The linear indicator has to be specified its width so that it could show up with the radial one.
- When I passed width and value, the linear indicator does not have the correct filled value.

I'm not sure what to do to correct the styling when having these together.

Reply:
Hmm, I think if you use `flex-direction: column` on the container, then it should work better? We shouldn't need to pass an explicit `width` to the progress indicators.

Reply:
`flex-direction: column` seems to have the same problem :(

Reply:
So there are two problems here:
1. `align-items: center` tries to collapse the items. To fix this, we can add `.demo-container > * { width: 100% }` in the CSS, instead of the inline style.
2. There is actually a bug in `ProgressLinear`: it does not correctly merge the `style` prop. We should fix this in a separate PR.
   https://github.com/iTwin/iTwinUI/blob/76613b5ffbe04a7ad899cd7c39a4dbc9dd0b99b1/packages/itwinui-react/src/core/ProgressIndicators/ProgressLinear.tsx#L85-L90

Reply:
I have added this as an after-PR todo. Can I use `:first-child` instead of grabbing all direct children? It caused a pretty odd visual error
![image](https://github.com/user-attachments/assets/467fd998-b429-4b53-9707-632dc9c4b2fd)

Reply:
`:first-child` is not reliable because the order of elements might change. Can you try removing `align-items: center` and centering using `.demo-container > * { margin-inline: auto }`?

Reply:
Hmm, I still have the same problem as above.

Reply:
Sorry I should have clarified: the `width: 100%` also needs to be removed. It was only necessary when `align-items: center` was used.

But it looks like `margin-inline: auto` reintroduces the problem where the linear indicator collapses entirely.

For the sake of simplicity, I would say let's just *not* center it. So this would be the _entire_ CSS:

```css
.demo-container {
  display: flex;
  flex-direction: column;
  width: 200px;
  gap: var(--iui-size-s);
}
```

Reply:
I have made the change.